src/main/java/cl/transbank/onepay/model/Transaction.java

```diff
@@ -18,15 +18,48 @@
     private static final String SEND_TRANSACTION = "sendtransaction";
     private static final String COMMIT_TRANSACTION = "gettransactionnumber";
 
+    /**
+     *
+     * @param cart
+     * @return
+     * @throws IOException
+     * @throws SignatureException
+     * @throws TransactionCreateException
+     *
+     * @deprecated use {@link #create(ShoppingCart, String)} instead
+     */
+    @Deprecated
     public static TransactionCreateResponse create(@NonNull ShoppingCart cart)
             throws IOException, SignatureException, TransactionCreateException {
-        return create(cart, null);
+        return create(cart, Onepay.DEFAULT_CHANNEL);
     }
 
+    public static TransactionCreateResponse create(@NonNull ShoppingCart cart, @NonNull String channel)
```

Comment:
I recommend we use an enum for the channel. That way the overloaded methods will be more obvious (remember that there will be an overloaded method to take a EUN and that's a String too)

Reply:
As a plus, we will avoid any mistyping errors with the use of the enum

Reply:
Done with enum now